core/src/main/scala/Manifest.scala

```diff
@@ -405,26 +405,17 @@ object Manifest {
       .map(l => Foldable[List].fold(l))
   }
 
-  private def addVersionToLoadbalancers(m: Manifest)(r: Github.Release): List[Loadbalancer] = {
-    val major = Version.fromString(r.tagName).map(_.toMajorVersion)
+  private def addVersionToLoadbalancers(m: Manifest)(r: Github.Deployment): List[Loadbalancer] = {
+    // NOTE(timperret): given `ref` might not actually be parsable as a `Version`, something
+    // needs to happen here that is a little more meaningful.
```

Comment:
It definitely won't always be the case right, especially if we're releasing as part of a PR/branch or for experimentation. We should definitely have some handling here for that.